cmd/src/actions_exec.go

```diff
@@ -559,6 +559,11 @@ query ActionRepos($query: String!) {
 	for _, repo := range reposByID {
 		repos = append(repos, repo)
 	}
+
+	if len(repos) == 0 && !*verbose {
+		yellow.Fprintf(os.Stderr, "WARNING: No repositories matched by scopeQuery\n")
```

Comment:
should that affect the exit code as well?

Reply:
I don't think so, but happy to be persuaded otherwise. It's not really a hard error

Reply:
No blocker, we can revisit that decision when we feel it would be beneficial